src/components/Sponsors.tsx

```diff
@@ -0,0 +1,92 @@
+import { SPONSORS, type Sponsor, type SponsorType } from '@/data/sponsors';
+import Image from 'next/image';
+import FancyRectangle from './FancyRectangle';
+
+type SponsorCardProps = Sponsor & { reverse?: boolean };
+function SponsorCard({ image, name, description, website, type, reverse }: SponsorCardProps) {
+    return (
+        <div
+            className={`flex flex-col items-stretch gap-5 ${reverse ? 'md:flex-row-reverse' : 'md:flex-row'}`}
+        >
+            <div className="flex h-64 shrink-0 items-center justify-center bg-white md:h-auto md:w-64">
+                <Image
+                    src={`/images/sponsors/${image}`}
+                    alt={`${name} Logo`}
+                    width={200}
+                    height={200}
+                />
+            </div>
+            <div>
+                <FancyRectangle colour="white" offset="8" rounded>
+                    <div className="space-y-2 rounded-xl bg-white p-4 text-black">
```

Comment:
The padding should be a bit bigger for larger screens, so do `p-4 md:p-6` instead.